src/components/Aside/AddRestaurantModal.jsx

```diff
@@ -1,42 +1,38 @@
 import Categories from "../../data/Category.js";
 import styles from "../../css/RestaurantModal.module.css";
+import Modal from "./Modal.jsx"
 
-function AddRestaurantModal() {
-    const categories = Categories();
+function AddRestaurantModal({ onChangeAddModal }) {
+    const categories = Categories().filter(
+        (category) => category.name !== "전체"
+    );
 
     return (
-        <div className={`${styles.modal} ${styles["modal--open"]}`}>
-            <div className={styles["modal-backdrop"]}></div>
-            <div className={styles["modal-container"]}>
-                <h2 className={`${styles["modal-title"]} text-title`}>새로운 음식점</h2>
-                <form>
-                    <div className={`${styles["form-item"]} ${styles["form-item--required"]}`}>
-                        <label htmlFor="category" className="text-caption">카테고리</label>
-                        <select name="category" id="category" required>
-                            <option value="">선택해 주세요</option>
-                            {categories.map((category) => (
-                                <option key={category.id} value={category.name}>{category.name}</option>
-                            ))}
-                        </select>
-                    </div>
-
-                    <div className={`${styles["form-item"]} ${styles["form-item--required"]}`}>
-                        <label htmlFor="name" className="text-caption">이름</label>
-                        <input type="text" name="name" id="name" required></input>
-                    </div>
-
-                    <div className={styles["form-item"]}>
-                        <label htmlFor="description" className="text-caption">설명</label>
-                        <textarea name="description" id="description" cols="30" rows="5"></textarea>
-                        <span className={`${styles["help-text"]} text-caption`}>메뉴 등 추가 정보를 입력해 주세요.</span>
-                    </div>
-
-                    <div className={styles["button-container"]}>
-                        <button className={`${styles.button} ${styles["button--primary"]} text-caption`}>추가하기</button>
-                    </div>
-                </form>
-            </div>
-        </div>
+        <Modal title="새로운 음식점" onClose={() => onChangeAddModal(null)}>
+            <form method="post" onSubmit={(event) => onChangeAddModal(event.target)}>
```

Comment:
이 부분에서 onChangeAddModal로 event 값을 전달하는 것 대신
onSubmit 함수를 App 에서 따로 만들어 관리하는 것은 어떨까요?

그리고 이건 사소한건데 onChangeAddModal 대신 close 하는 것이 명확하니 handleCloseModal로 함수명을 변경하는 것도 생각해보시면 좋을 거 같아요!
```suggestion
            <form method="post" onSubmit={(event) => onFormSubmit(event.target)} >
```

Reply:
값 전달과 모달 창을 닫는 것을 분리하시라는 말씀이신거죠??
좋은 방법 같네요!!